src/data/faq.json

```diff
@@ -1162,7 +1161,7 @@
                                     "If you did not receive the test result through the Corona-Warn-App, please contact your local health care professional, testing center, or health department. Even if your positive test result is not available in the Corona-Warn-App, but you have been notified of a positive test result outside of the Corona-Warn-App, you can still warn others via the app. For more details, please see the FAQ: <a href='#warn_for_others' target='_blank'>How can I warn for others?</a>",
                                     "</ul>",
                                     "<b>Can multiple tests be registered?</b>",
-                                    "You can scan one personal PCR and one rapid test. Each QR code for a test can only be scanned on one device. If the test result is subsequently deleted, it can be recovered from the recycle bin for 30 days, but cannot be scanned on another device. This is to prevent misuse and false positives. You can find more information here: <a href ='#QRcodes' target='_blank'>What to do with several PCR-Test QR codes?</a>",
+                                    "You can scan one personal PCR and one rapid test per family member. After scanning a test, select whether it is intended for you or for a family member. The status of a test, regardless of whether it belongs to you or a family member, can subsequently be called up on the status tab. Each QR code for a test can only be scanned on one device. If the test result is subsequently deleted, it can be recovered from the recycle bin for 30 days, but cannot be scanned on another device. This is to prevent misuse and false positives. You can find more information here: <a href ='#QRcodes' target='_blank'>What to do with several PCR-Test QR codes?</a>",

```

Comment:
I don't understand this. It says that you can scan one personal PCR and one rapid test per family member. This means, that the use case e.g. scanning one PCR test for a family member and a personal RAT is not possible at the same time. Is this true?

I think this is an incorrect translation from German.

Reply:
Changed to "Each family member can scan one PCR test and one rapid test.", thanks!